DOCS: Minor changes to quickstart docs

andrew-oc opened the pull request:
### Description

Changed usage of FixedUpdate to Update so users don't start using wasPressedThisFrame in FixedUpdate until they know more about the repercussions

### Checklist

Before review:

- [x] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - FogBugz ticket attached, example `([case %number%](https://issuetracker.unity3d.com/issues/...))`.
    - FogBugz is marked as "Resolved" with *next* release version correctly set.

andrew-oc committed:
Minor changes to quickstart docs

* changed usage of FixedUpdate to Update so users don't start using wasPressedThisFrame in FixedUpdate until they know more about the repurcussions
* added a more comprehensive TOC

andrew-oc committed:
Minor changes to quickstart docs

* changed usage of FixedUpdate to Update so users don't start using wasPressedThisFrame in FixedUpdate until they know more about the repurcussions
* added a more comprehensive TOC

Commit:
FIX: Incorrect docs RE WebGL touch support (#1389).

ekcoh reviewed (approved):
Looks good to me, but seems like branch needs an update prior to merge.

NibbleByte committed:
FIX: Incorrect binding display strings for TwoModifiersComposite (#1372).

Commit:
CHANGE: Clamp defaultButtonPressPoint to 0.0001 (case 1349002, #1391).

antoine-agthe-unity committed:
FIX: InputEventTrace skipping over empty frames (#1368).

Commit:
CHANGE: Rework focus handling (#1324).

mattdatwork committed:
CHANGE: XR define changes (#1350)

andrew-oc committed:
Merge branch 'fix-quickstart-documentation' of github.com:Unity-Technologies/InputSystem into fix-quickstart-documentation

andrew-oc committed:
Fix docfx bug

Removed a link from a header item because docfx was generating an invalid anchor
